mw-webapp/src/dataAccessLogic/UserPreviewDAL.ts

```diff
@@ -10,11 +10,18 @@ import {PartialWithUuid} from "src/utils/PartialWithUuid";
  */
 export class UserPreviewDAL {
 
+  /**
+   * Get amount of all users in collection
+   */
+  public static async getUsersPreviewAmount(): Promise<number> {
+    return await UserService.getUsersDTOAmount();
+  }
+
   /**
    * Get all UserPreview
    */
-  public static async getUsersPreview(): Promise<UserPreview[]> {
-    const usersDTO = await UserService.getUsersDTO();
+  public static async getUsersPreview(uuid?: string): Promise<UserPreview[]> {
+    const usersDTO = uuid ? await UserService.getUsersDTOPart(uuid) : await UserService.getUsersDTO();
```

Comment:
not clear what happened here? why 2 methods?

Reply:
Which 2 methods?

Reply:
Get rid of one of them

Reply:
Deleted second method